pkgs/development/python-modules/minimal-snowplow-tracker/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, requests
+, six
+}:
+
+buildPythonPackage rec {
+  pname = "minimal-snowplow-tracker";
+  version = "0.0.2";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-rKv3Vy2w5/XL9pg9SV7vVAgfcb45IzDrOq25zLOdqqQ=";
+  };
+
+  propagatedBuildInputs = [
+    requests
+    six
+  ];
+
```

Comment:
Do we have tests to run?

Reply:
The source code has some tests that might have been left over from when it was forked, but the repo has no tags of its own to mark its releases, and the tests are not included in the PyPI release. I disabled `doCheck` and left a comment saying that there are no tests.